# Define the goal
string robot_name
---
# Define the result
bool found
---
# Define a feedback message
bool progress